cmd/gpu-kubelet-plugin/device_state.go

```diff
@@ -150,12 +150,12 @@ func (s *DeviceState) Prepare(ctx context.Context, claim *resourceapi.ResourceCl
 		return nil, fmt.Errorf("unable to create CDI spec file for claim: %w", err)
 	}
 
-	preparedClaims[claimUID] = preparedDevices
+	preparedDevicesByClaimUID[claimUID] = preparedDevices
 	if err := s.checkpointManager.CreateCheckpoint(DriverPluginCheckpointFile, checkpoint); err != nil {
 		return nil, fmt.Errorf("unable to sync to checkpoint: %v", err)
 	}
 
-	return preparedClaims[claimUID].GetDevices(), nil
+	return preparedDevicesByClaimUID[claimUID].GetDevices(), nil
```

Comment:
Has this changed between where we set it on line 153 and here? Is returning `preparedDevices.GetDevices()` clearer?

Reply:
My edit here was rather ignorant. I see. I think we can do what you propose. Will report back.